README.md

```diff
@@ -58,7 +58,7 @@ Click on `Add notebooks` (upper right) to create a notebook.
 * Use the `Spark Service` pulldown to select your `DSX-Spark` service.
 * Click the `Create Notebook` button.
 
-![](doc/source/images/create_notebook.png)
+![](docs/sources/create_notebook.png)
```

Comment:
I think Steve has us consistently following the doc/source/images tree convention